recipes/xsv/meta.yaml

```diff
@@ -0,0 +1,32 @@
+{% set version = "0.13.0" %}
+
+package:
+  name: xsv
+  version: {{ version }}
+
+source:
+  url: https://github.com/BurntSushi/xsv/archive/{{ version }}.tar.gz
+  sha256: 2b75309b764c9f2f3fdc1dd31eeea5a74498f7da21ae757b3ffd6fd537ec5345
+
+build:
+  number: 0
+
+requirements:
+  host:
+    - rust
+
+test:
+  commands:
+    - xsv --help > /dev/null
+
+about:
+  home: https://github.com/BurntSushi/xsv
+  license: MIT
+  license_file: LICENSE-MIT
+  summary: A fast CSV toolkit written in Rust.
+
+extra:
+  recipe-maintainers:
+    - bgruening
+    - rvalieris
+    - johanneskoester
```

Comment:
Does everyone agree to be a maintainer?

@bgruening , @johanneskoester

Reply:
it should be fine, I can remove them later if they want right ?

Reply:
Ok with me. Johannes does not follow conda-forge but is generally ok I think :)